src/ui/public/timepicker/toggle.html

```diff
@@ -1,33 +1,33 @@
 <ul ng-show="timefilter.enabled" class="nav navbar-nav navbar-right navbar-timepicker">
   <li>
-    <a
+    <button
       ng-click="toggleRefresh()"
       ng-show="timefilter.refreshInterval.value > 0">
-
       <i class="fa" ng-class="timefilter.refreshInterval.pause ? 'fa-play' : 'fa-pause'"></i>
-    </a>
+    </button>
   </li>
 
   <li
-    ng-class="{active: pickerTemplate.is('interval') }"
-    ng-show="timefilter.refreshInterval.value > 0 || !!pickerTemplate.current"
+    ng-show="timefilter.refreshInterval.value > 0 || configTemplate.is('interval') || configTemplate.is('filter')"
     class="to-body">
 
-    <a ng-click="pickerTemplate.toggle('interval')" class="navbar-timepicker-auto-refresh-desc">
+    <button ng-click="configTemplate.toggle('interval')"
+    ng-class="{active: configTemplate.is('interval') }"
+    class="navbar-timepicker-auto-refresh-desc">
       <span ng-show="timefilter.refreshInterval.value === 0"><i class="fa fa-repeat"></i> Auto-refresh</span>
       <span ng-show="timefilter.refreshInterval.value > 0">{{timefilter.refreshInterval.display}}</span>
-    </a>
+    </button>
 
   </li>
 
-  <li class="to-body" ng-class="{active: pickerTemplate.is('filter')}">
-    <a
-      ng-click="pickerTemplate.toggle('filter')"
+  <li class="to-body" >
```

Comment:
We can get rid of everything to a `to-body` class. It was used to draw the little arrow on tabs in the right color, but we don't have those anymore.